source/includes/usage-examples/connect-sample-app.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+// Start example code here
+
+// End example code here
+
+$admin = $client->admin;
+$result = $admin->command(['ping' => 1]);
```

Comment:
```suggestion
try {
    $client->test->command(['ping' => 1]);
    echo 'Successfully pinged the MongoDB server.', PHP_EOL;
} catch (MongoDB\Driver\Exception\RuntimeException $e) {
    printf("Failed to ping the MongoDB server: %s\n", $e->getMessage());
}
```

There's no need to use the `admin` database, so I replaced it with `test`. The `Database::command()` helper either returns a cursor or throws an exception, so checking `$result` with an if/else wasn't correct. The cursor isn't relevant to us, so we can just print a success message after invoking the method.

Per the [exception class tree](https://www.php.net/manual/en/mongodb.exceptions.tree.php), a RuntimeException from the extension is the best top-level exception to use here, as it will catch both client-side network errors and any server-side command errors.